packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -48,13 +48,15 @@ contract FundingRateApplier is Testable, Lockable {
     // If another 1% fee is charged, the multiplier should be 1.01^2 (1.0201).
     FixedPoint.Unsigned public cumulativeFundingRateMultiplier;
 
+    Timer timer;
+
     /****************************************
      *                EVENTS                *
      ****************************************/
 
-    // TODO: Decide which params to emit in this event.
     event NewFundingRate(
         uint256 indexed newMultiplier,
+        uint256 lastUpdateTime,
         uint256 indexed updateTime,
         uint256 indexed paymentPeriod,
         uint256 latestFundingRate,
```

Comment:
nit: Maybe we make `latestFundingRate` indexed as it seems more informative.

Reply:
sure